client/gutenberg/extensions/map-block/add-point/index.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Wordpress dependencies
+ */
+
+import { __ } from '@wordpress/i18n';
+
+import { Component } from '@wordpress/element';
+
+import {
+	Button,
+	Dashicon,
+	Popover
+} from '@wordpress/components';
+
+/**
+ * External dependencies
+ */
+
+/**
+ * Internal dependencies
+ */
+
+import LocationSearch from '../location-search';
+
+import './style.scss';
+
+export class AddPoint extends Component {
+
+	constructor() {
+		super( ...arguments );
+		this.onAddPoint = this.onAddPoint.bind( this );
+	}
+
+	onAddPoint( point ) {
+
+		this.props.onAddPoint( point );
+
+	}
+
+	render() {
+		const {
+			onClose,
+			onAddPoint
+		} = this.props;
+		return (
+			<Button
+				className='map__add_btn'
+			>
+				Add point
```

Comment:
Let's make sure to localize all the copy.

Reply:
Got this in https://github.com/Automattic/wp-calypso/commit/11702a5987ce8bc9503ddc5eb0a180ee50fe9aec